docs-conceptual/azureadps-2.0/using-extension-attributes-sample.md

```diff
@@ -140,3 +139,6 @@ If you no longer need an extension property, you can delete it:
 ```powershell 
 Remove-AzureADApplicationExtensionProperty -ObjectId (Get-AzureADApplication -SearchString "My Properties Bag").ObjectID -ExtensionPropertyId 91ec8ae5-6813-4453-afd7-31680a484892 
 ```
+
+> [!NOTE]
+> You can neither remove the attribute from Azure AD once extended nor delete the `Tenant Schema Extension App` without raising a request to the PayOps Team. Also, turning off the directory extensions option from Azure AD wizard won't remove the attributes or let you get rid of the app.
```

Comment:
Should we use "Entra ID" rather than Azure AD?

Reply:
Thanks, @dkershaw10. The update on docs to reflect the new naming will start after August 15. They gave a month from the date of the announcement.